app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -302,6 +306,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         reasonList.add(getString(R.string.deletion_reason_not_interesting));
         reasonList.add(getString(R.string.deletion_reason_no_longer_want_public));
         reasonList.add(getString(R.string.deletion_reason_bad_for_my_privacy));
+        //add corresponding mappings in english locale so that we can upload it in deletion request
```

Comment:
Stylistic changes: `// Add corresponding mappings in English locale so that we can upload it in the deletion request.` (space after `//`, sentence uppercase and final point, uppercase for language name). Thanks! :-)